go/announcer.go

```diff
@@ -49,14 +50,15 @@ InterfacesLoop:
 		}
 	}
 	cfg := dnssd.Config{
-		Name:   "RPM Test Server",
+		Name:   fmt.Sprintf("RPM Test Server(%d)", port),
```

Comment:
Nice!

Reply:
Ultimately I'm not sure this is best here, especially when it comes to multiple listeners/ports/protocols. But I'll kick that can down the road.